addons/account/models/account_journal.py

```diff
@@ -798,3 +781,17 @@ def _get_last_bank_statement(self, domain=None):
         last_statement_domain = (domain or []) + [('journal_id', '=', self.id)]
         last_st_line = self.env['account.bank.statement.line'].search(last_statement_domain, order='date desc, id desc', limit=1)
         return last_st_line.statement_id
+
+    def _ensure_payment_method_type_on_journal(self, payment_method_type):
+        for journal in self:
```

Comment:
Not sure this method is really helpful. Making ```
bank_journal.outbound_payment_method_id = [Command.create({'method_type': <my_method>.id})]
``` is cleaner and less spaghetti in `_enable_check_printing_on_bank_journals` imo